index.ts

```diff
@@ -740,11 +739,14 @@ commander
       open(url);
       process.exit(0);
     }
-    const logger = new logging({
-      projectId,
-    });
-    return logger.getEntries().then(printLogs).catch((err: any) => {
-      console.error(ERROR.LOGS_UNAVAILABLE);
+    getAPICredentials(async () => {
+      const logger = google.logging({version: 'v2', auth: oauth2Client}) as Logging;
+      const { data } = await logger.entries.list({
+        'resourceNames': [
```

Comment:
Don't quote object keys.
https://palantir.github.io/tslint/rules/object-literal-key-quotes/

`object-literal-key-quotes: 'as-needed'` can be added to `tslint.json` to prevent this.